backend/src/tasks/sslyze.ts

```diff
@@ -0,0 +1,41 @@
+import { CommandOptions } from './ecs-client';
+import { getLiveWebsites, LiveDomain } from './helpers/getLiveWebsites';
+import PQueue from 'p-queue';
+import { execSync } from 'child_process';
+
+const sslyze = async (domain: LiveDomain): Promise<void> => {
+  try {
+    const path = `/tmp/${domain.name}.crt`;
+    const res = execSync(
+      `echo "" | openssl s_client -connect ${domain.name}:443 > ${path}; openssl x509 -in ${path} -noout -enddate`,
```

Comment:
hope this isn't too bad security-wise... I can rewrite if you think it's better to do so

Reply:
Yeah, let's break it into arguments instead (e.g. https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) so it's not possible to inject into the command

Reply:
The concern is a domain name like `;ls;` would allow executing arbitrary commands